src/http-api/index.js

```diff
@@ -23,6 +26,11 @@ function HttpApi (repo, config) {
   this.log = debug('jsipfs:http-api')
   this.log.error = debug('jsipfs:http-api:error')
 
+  // Setup debug metrics collection
+  const collectDefaultMetrics = prometheusClient.collectDefaultMetrics
+  collectDefaultMetrics({ timeout: 5000 })
+  prometheusGcStats(prometheusClient.register)()
```

Comment:
@diasdavid "phone home" as in send the data somewhere? No, it'll only be in the local node, available to collect when hitting the endpoint. The data is not sent somewhere.